ldap/servers/plugins/replication/cl5_clcache.c

```diff
@@ -376,16 +369,16 @@ clcache_load_buffer(CLC_Buffer *buf, CSN **anchorCSN, int *continue_on_miss, cha
         buf->buf_state = CLC_STATE_READY;
         if (anchorCSN)
             *anchorCSN = buf->buf_current_csn;
-        rc = clcache_load_buffer_bulk(buf, flag);
+        rc = clcache_load_buffer_bulk(buf, dbop);
 
-        if (rc == DB_NOTFOUND && continue_on_miss && *continue_on_miss) {
+        if (rc == DBI_RC_NOTFOUND && continue_on_miss && *continue_on_miss) {
             /* make replication going using next best startcsn */
             slapi_log_err(SLAPI_LOG_ERR, buf->buf_agmt_name,
-                          "clcache_load_buffer - Can't load changelog buffer starting at CSN %s with flag(%s). "
+                          "clcache_load_buffer - Can't load changelog buffer starting at CSN %s with operation(%s). "
                           "Trying to use an alterantive start CSN.\n",
                           (char *)buf->buf_key.data,
-                          flag == DB_NEXT ? "DB_NEXT" : "DB_SET");
-            rc = clcache_load_buffer_bulk(buf, DB_SET_RANGE);
+                          dblayer_op2str(dbop));
```

Comment:
indentation 2

Reply:
I do not see any indentation problem here. Can you precise  ?

Reply:
dblayer_op2str() to very far to the right.  It should be aligned with the "rc =" :-)

Reply:
I think that alignment is correct:
dblayer_op2str is not the start of an instruction but the last parameter of slapi_log_err call
    It is aligned with the other parameters (as flag == DB_NEXT ?  ... was)

Reply:
Oh, you're right.  My bad...